### PR TITLE
add config comment in yolov3_darknet.yml

### PR DESCRIPTION
**Pull request by heavengate**
**add config comment in yolov3_darknet.yml**
1. generate full config file in docs/config_example/yolov3_darknet.yml
2. add config comment for all configs

### CODE REVIEW

File: PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+DarkNet:
+  # batch normalization type in training, sync_bn for synchronized batch normalization
+  norm_type: sync_bn
+  # L2 weight decay factor of batch normalization layer
+  norm_decay: 0.
+  # Darknet convolution layer number, only support 53 currently
+  depth: 53
+
+# YOLOv3 head module
+# generate bbox outpur in evaluation and calculate loss in training
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml]: output

File: PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+# fluid.layers.yolov3_loss / fluid.layers.yolo_box
+YOLOv3Head:
+  # anchor mask of 3 yolo_loss/yolo_box layer, each yolo_loss/yolo_box layer has 3 anchors
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml]: layers

File: PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+  anchor_masks: [[6, 7, 8], [3, 4, 5], [0, 1, 2]]
+  # 9 anchors for 3 yolo_loss/yolo_box layer, generated by perform kmeans on COCO gtboxes
+  anchors: [[10, 13], [16, 30], [33, 23],
+            [30, 61], [62, 45], [59, 119],
+            [116, 90], [156, 198], [373, 326]]
+  # L2 weight decay factor of batch normalization layer
+  norm_decay: 0.
+  # ignore threshold for yolo_loss layer
+  ignore_thresh: 0.7
+  # whether use label smooth in yolo_loss layer, it is recommend to set as true when only num_classes is very big
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml]: recommended

File: PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+  label_smooth: true
+  # fluid.layers.multiclass_nms
+  # non-max suppress for output prediction boxes, see multiclass_nms for following parameters
```
Comment (jerrywgz) [inline PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml]: 需要简单描述下nms的操作和主要参数的含义

File: PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml (hunk 1 above)
Comment (heavengate) [reply PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml]: Done, thanks!

File: PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml (hunk 2 above)
Comment (heavengate) [reply PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml]: Done, thanks!

File: PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml (hunk 3 above)
Comment (heavengate) [reply PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml]: Done, thanks!

File: PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml (hunk 4 above)
Comment (heavengate) [reply PaddleCV/PaddleDetection/docs/config_example/yolov3_darknet.yml]: Done, thanks!

**Review by jerrywgz — COMMENTED** [review]
加上log_iter吧，默认20

**Comment by heavengate** [comment]
> 加上log_iter吧，默认20

Done, thanks!

**Review by jerrywgz — APPROVED** [review]
LGTM

**Pull request merged**